packages/extension/src/components/pages/Nfts/Nfts.tsx

```diff
@@ -0,0 +1,51 @@
+import { FC, useContext, useEffect, useState } from 'react';
+
+import { AccountNFTokenResponse } from '@gemwallet/constants';
+
+import { LedgerContext } from '../../../contexts';
+import { NftListing } from '../../organisms/NftListing';
+import { PageWithHeader } from '../../templates';
+
+const initalState = {
+  account_nfts: [],
+  marker: null
+};
+
+export const Nfts: FC = () => {
+  const { getNFTs } = useContext(LedgerContext);
+
+  const [nfts, setNfts] = useState<AccountNFTokenResponse>(initalState);
+
+  const fetchNfts = async () => {
+    try {
+      const payload = {
+        limit: 20,
+        marker: nfts.marker
+      };
+
+      if (!nfts.marker) {
+        delete payload.marker;
+      }
+
+      const response = await getNFTs(payload);
```

Comment:
I would simplify this code that way:

```suggestion
      const response = await getNFTs({
            limit: 20,
            marker: nfts.marker ?? undefined
      );
```

Reply:
added